connectivity_summary.py

```diff
@@ -46,7 +52,10 @@ def arg_to_date(arg):
     print("No experiments loaded; bailing out.")
     sys.exit(-1)
 
-expts = all_expts.select(start=args.start, stop=args.stop, region=args.region)
+for i, ex in enumerate(all_expts._expts):
```

Comment:
then the next line:
``expts = all_expts.select(....)``

would need to be changed to:
 `expts = ex.select(...)`

correct?